src/index.html

```diff
@@ -11,6 +11,39 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="container">
+
+      <form
+      class="search_bar"
+      data-qa="big"
+      action="#"
+      method="get">
+        <div class="input-wrapper__first">
+          <img class="input-img" src="images/Search.svg"
+          alt="Search-image">
```

Comment:
Need to remove it, because you need to add image as the background for the input
```suggestion
```